homeassistant/components/yeelight/__init__.py

```diff
@@ -218,41 +213,55 @@ def available(self):
         """Return true is device is available."""
         return self._available
 
+    @property
+    def model(self):
+        """Return configured device model."""
+        return self._model
+
     @property
     def is_nightlight_enabled(self) -> bool:
         """Return true / false if nightlight is currently enabled."""
         if self.bulb is None:
             return False
 
-        return self.bulb.last_properties.get('active_mode') == '1'
+        return self._active_mode == '1'
```

Comment:
Maybe define `ACTIVE_MODE_NIGHTLIGHT = '1'` and use the constant here for readability. What are the other possible modes and should they also be handled somehow?

Reply:
From yeelight docs:

0: daylight mode / 1: moonlight mode (ceiling light only) . I don't think we need to handle anything more now. Added suggested const.